samples/sensor/bme680/README.rst

```diff
@@ -38,6 +38,11 @@ In this example below the :ref:`nrf52840dk_nrf52840` board is used.
    :board: nrf52840dk_nrf52840
    :goals: build flash
 
+Example command to builf for E7 board (RTSS_HE devkit):
+
+.. code-block::
+   west build -p auto -b alif_e7_dk/ae722f80f55d5xx/rtss_he ../alif/samples/sensor/bme680 -S alif-dk
```

Comment:
We shouldn't be adding build instructions to all samples READMEs. Unnecessary duplication of instructions.

Reply:
A correction:

Please follow the same style as upstream Zephyr when it comes to documenting the sample build instructions:

    https://github.com/alifsemi/zephyr_alif/blob/112ca2f246b717a67ae7d3b2c6c34a29f58d92dc/samples/bluetooth/bap_unicast_server/README.rst?plain=1#L24-L38